### PR TITLE
[Storage] rename SKUs to Skus

### PR DESCRIPTION
**Pull request by JasonYang-MSFT**
## Description

Fixed bad names in swagger for python SDK, which affect other SDKs.
Since 7.0.0 haven't released yet, this will not break current experience for customers.

https://github.com/Azure/azure-rest-api-specs/pull/1650

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please add < PackageReleaseNotes > and add Release notes in .csproj. This helps in knowing what is included in the nuget package.
I would recommend to use < ![CDATA and then format your Release notes

**Comment by JasonYang-MSFT** [comment]
Changelog added, it should be available once this pr is merged.

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj [hunk 1]
```diff
@@ -10,4 +10,5 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <PackageReleaseNotes>See https://aka.ms/asdotnetsdkchangelog for release notes.</PackageReleaseNotes>
```
Comment (shahabhijeet) [inline src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: This property has to be part of a propertygroup

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj (hunk 1 above)
Comment (JasonYang-MSFT) [reply src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: Updated.

**Pull request merged**